### PR TITLE
heimer: bump version

### PR DESCRIPTION
**Pull request by mazbrili**

### CODE REVIEW

File: app-office/heimer/heimer-1.20.0.recipe [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+	devel:libQt5Gui$secondaryArchSuffix
+	devel:libQt5Network$secondaryArchSuffix
+	devel:libQt5PrintSupport$secondaryArchSuffix
+	devel:libQt5Svg$secondaryArchSuffix
+	devel:libQt5Xml$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:gcc$secondaryArchSuffix
+	cmd:ld$secondaryArchSuffix
```
Comment (Begasus) [inline app-office/heimer/heimer-1.20.0.recipe:62]: Missing:
```diff
+	cmd:lrelease$secondaryArchSuffix >= 5
```

**Comment by Begasus** [comment]
Maybe remove the previous recipe while you are in the process of updating?

**Comment by extrowerk** [comment]
Nice job, thanks for the recipe, i planned since long to write one, but you was faster. Great!

**Comment by Begasus** [comment]
Pokes @mazbrili ... could you add the missing dependency? (when checked on 64 bit)

File: app-office/heimer/heimer-1.20.0.recipe (hunk 1 above)
Comment (mazbrili) [reply app-office/heimer/heimer-1.20.0.recipe:62]: done thanks @Begasus

**Comment by mazbrili** [comment]
> Maybe remove the previous recipe while you are in the process of updating?

ok. this done too

**Comment by mazbrili** [comment]
> Pokes @mazbrili ... could you add the missing dependency? (when checked on 64 bit)

ok done.. sorry i'm not on access to my machine this couple day.. so i late to respond all of you @Begasus  @extrowerk

**Comment by mazbrili** [comment]
thanks all

**Pull request merged**